vitrina/datasets/views.py

```diff
@@ -42,3 +42,20 @@ def get_context_data(self, **kwargs):
         }
         context_data.update(extra_context_data)
         return context_data
+
+
+class DatasetMembersView(ListView):
+    model = DatasetMembers
+    template_name = 'vitrina/datasets/members_list.html'
+    context_object_name = 'dataset_members'
+    paginate_by = 5
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        context['org'] = Organization.objects.get(kind=self.kwargs.get('org_kind'),
+                                                  slug=self.kwargs.get('org_slug'))
+        context['members'] = DatasetMembers.objects.filter(organization=context['org'], contact=True)
```

Comment:
Please move this to [get_queryset](https://docs.djangoproject.com/en/3.2/ref/class-based-views/mixins-multiple-object/#django.views.generic.list.MultipleObjectMixin.get_queryset). And the query should look like this:

```suggestion
        dataset = Dataset.objects.get_from_url_args(**self.kwargs)
        return dataset.dataset_member_set.all()
```

You need to implement `get_from_url_args` on the default Dataset model manager.

- https://docs.djangoproject.com/en/3.2/topics/db/managers/
- https://docs.djangoproject.com/en/3.2/ref/models/fields/#foreignkey